_includes/nav.md

```diff
@@ -9,11 +9,10 @@
     <li class="sub">
       <input type="checkbox" name="nav-radio" id="guide-radio"><label for="guide-radio">Guides</label>
       <ul>
+      <li><a href="/guides/setup">Download &amp; Setup</a></li>
```

Comment:
Yes. Getting started is the part where you learn to write your first templates and such. Should assume that you already have Dust ready to use.